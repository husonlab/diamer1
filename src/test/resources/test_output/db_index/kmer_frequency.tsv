frequency	count
1	1320
2	1
3	1
