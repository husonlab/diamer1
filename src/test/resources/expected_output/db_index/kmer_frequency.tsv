frequency	count
1	1307
2	1
3	1
